scalyr_agent/monitor_utils/k8s.py

```diff
@@ -1151,6 +1300,11 @@ def get_container_runtime( self ):
         result = None
         self._lock.acquire()
         try:
+            if self._state.cache_config.use_controlled_warmer:
+                # TODO: For now, controlled_warmer usage directly implies/requires 'docker' runtime
```

Comment:
There are some outstanding TODOs that must be resolved before merging back to master. This is one of them.

Reply:
Change to support other runtimes. 
Query path outside rate limiter. Need to rate limit.?